src/headline_grabber/configurations/sites/tol.yaml

```diff
@@ -0,0 +1,17 @@
+abbreviation: tol
+name: The Times of London
+url: https://www.thetimes.com
+engine: beautifulsoup
+selectors:
+    headline:
+      tag: div
+      identifier: Item-content
+    link:
+      tag: a
+      identifier: href
+    title:
+      tag: h3
+      identifier: Item-headline
+    description:
+      tag: p
+      identifier: Dip Item-dip
```

Comment:
Let's add a test or two confirming the integration is working, see @tonthattuit's recent PR for testing examples

Reply:
Three tests were added.
- One to verify the get headlines work with beautiful soup.
- One to verify the html parser works with a prewritten html file.
- One to verify the pipeline_context works with the yaml setup